lib/customer.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'restaurant'
+
+class Customer
+  attr_reader :name, :order, :restaurant
+
+  def initialize(restaurant = Restaurant.new)
+    @restaurant = restaurant
+    @order = Hash.new(0)
+  end
+
+  def menu
+    restaurant.view_menu
+  end
+
+  def place_order(dish, quantity = 1)
+    raise "Please select a dish from the menu." if !restaurant.menu_include?(dish)
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)